docs/changelog_for_devs.md

```diff
@@ -44,6 +44,9 @@ APIs/RPC interface.
 
   For details, please refer to the `README.md` and the in-file documentation.
 
+- Modify event `MarketDisputed` to have an additional field for the disputant.
```

Comment:
Could you add a link to the relevant PR (like it's done further down).

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1148/commits/f14c9fdaf9c964e04730e0bd5e299cc26f568dfe